src/client/styles/hello.scss

```diff
@@ -1,5 +1,27 @@
-$color: blue;
+$color: #222;
+$textColor: #FFF;
+$linkColor: #09F;
 
 body {
   background-color: $color;
+  color: $textColor
+}
+
+.header {
+	text-align: center;
+}
+
+#content > div {
+	text-align: center;
+}
+
+a {
+	color: $linkColor;
+	text-decoration: none;
+
+}
+
+li {
+	display: inline-block;
+	margin-right: 1rem;
```

Comment:
General rule of thumb: Move any and all colors and/or sizes to vars, unless it's the number 0. So, even this tiny `1rem` margin shift here would be better served in a variable, which I'd probably just name `$li-margin-right` for simplicity.

By moving all of these values to vars, we make it easy to tweak later, especially for a designer. But also, by moving these up it makes it super easy to recognize design patterns that we can then generalize/abstract out to external variable files.